src/jobs/helm_aws.yml

```diff
@@ -106,7 +106,10 @@ steps:
             cmd_name: Reset the replica count of each deployment for the specified env
             deploy: false
             cmd: |
-              if [[ "<< parameters.reset_dev_env_replica_count >>" =~ ^dev-env.* ]]; then
+              if [[ "<< parameters.aws_profile >>" == "production" ]]; then
+                echo "The AWS profile is set to 'production'. Exiting..."
+                exit 1
+              else
                 RELEASES=$(helm ls -n << parameters.reset_dev_env_replica_count >> -o json | jq -r '.[].name' | grep -v 'ingress\|ragnaros\|codacy-seed')
```

Comment:
In order to avoid changing this in all of our projects (at least in the first iteration), the name will be a bit misleading, because the parameter will be called "reset_dev_env_replica_count", but now can be used for the "codacy" namespace as well.

Further iterations can fix this if necessary.